st2api/st2api/controllers/v1/chatops.py

```diff
@@ -0,0 +1,95 @@
+# Licensed to the StackStorm, Inc ('StackStorm') under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import pecan
+import six
+
+from mongoengine import ValidationError
+from st2api.controllers import resource
+from st2common import log as logging
+from st2common.exceptions.apivalidation import ValueValidationException
+from st2common.models.api.action import ActionAliasAPI
+from st2common.persistence.actionalias import ActionAlias
+from st2common.models.api.base import jsexpose
+from st2common.rbac.types import PermissionType
+from st2common.rbac.decorators import request_user_has_permission
+from st2common.rbac.decorators import request_user_has_resource_api_permission
+from st2common.rbac.decorators import request_user_has_resource_db_permission
+from st2common.util.alias_matching import (list_format_strings_from_aliases,
+                                           match_command_to_alias)
+
+
+http_client = six.moves.http_client
+
+LOG = logging.getLogger(__name__)
+
+
+class ChatopsController(resource.ContentPackResourceController):
```

Comment:
Is it possible to keep those methods / operations on the alias controller?

The reason is that right now StackStorm concept is called action alias and the code doesn't use "ChatOps" term anywhere else (afaik) so it might be a bit confusing to call it ChatOps here...